vmdb/app/models/service_template_orchestration.rb

```diff
@@ -1,6 +1,16 @@
 class ServiceTemplateOrchestration < ServiceTemplate
   include ServiceOrchestrationMixin
 
+  before_save :remove_invalid_resource
+
+  def remove_invalid_resource
+    service_resources.each { |r| r.delete unless r.resource(true) }
+  end
+
+  def call_test(arg)
+    puts arg
+  end
```

Comment:
I'm ok keeping this if you add tests, but it should probably be removed.  :stuck_out_tongue_winking_eye: :trollface: